CHANGELOG.md

```
@@ -1,6 +1,35 @@
 Changelog
 =========
 
+Version 0.12.11
+---------------
+
+In this release, NullAway checks the requirement that JSpecify mode is only run on a compatible `javac` version /
+configuration, and fails if it detects an incompatibility.
+See <https://github.com/uber/NullAway/wiki/JSpecify-Support#supported-jdk-versions> for details.
+
+## What's Changed
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)